packages/cli/src/lib/project/project.ts

```diff
@@ -1,13 +1,21 @@
-import {createWriteStream, existsSync, unlinkSync, writeFileSync} from 'fs';
-import {join} from 'path';
+import {
+  createWriteStream,
+  existsSync,
+  readdirSync,
+  unlinkSync,
+  writeFileSync,
+} from 'fs';
+import {extname, join} from 'path';
 import {cli} from 'cli-ux';
 import archiver from 'archiver';
 import {InvalidProjectError} from '../errors';
 import extract from 'extract-zip';
 import {DotFolder, DotFolderConfig} from './dotFolder';
+import {readJsonSync, writeJsonSync, WriteOptions} from 'fs-extra';
 
 export class Project {
   private static readonly resourceFolderName = 'resources';
+  public static readonly jsonFormat: WriteOptions = {spaces: '\t'};
```

Comment:
![image](https://user-images.githubusercontent.com/12199712/128685275-39c9f759-9657-4ca9-9c66-a238b838d961.png)

Reply:
I don't think this will be enough to enforce a JSON format

Reply:
Why so, can you elaborate?

Reply:
As shown in the screenshot above, the JSONs (the one from the project and the other coming from the API) could have more than just space and tabs differences.

Reply:
Hm 👍 to keep a trace of what we discuss, I think it's not a problem 'cause what is the true goal here is to enforce formating in the expanded preview, which should be done here I reckon